tools/mmp/driver.cs

```diff
@@ -442,6 +442,29 @@ static void Main2 (string [] args)
 
 			}
 
+			if (IsUnified) {
```

Comment:
We should exclude Modern/Mobile as we do not want to break a profile by breaking some.
The alternative would be testing with Modern/Mobile - but that seems more resource intensive (and not as safe)